install/install_requirements.sh

```diff
@@ -94,27 +87,10 @@ LINUX_REQUIREMENTS_TO_INSTALL=(
     "${REQUIREMENTS_TO_INSTALL[@]}"
 )
 
-PLATFORM=$(uname -s)
-
-# Install torchtune and torchao requirements for Linux systems using nightly.
-# For non-Linux systems (e.g., macOS), install torchao from GitHub since nightly
-# build doesn't have macOS build.
-# TODO: Remove this and install nightly build, once it supports macOS
-if [ "$PLATFORM" == "Linux" ];
-then
-  (
-    set -x
-    $PIP_EXECUTABLE install --pre --extra-index-url "${TORCH_NIGHTLY_URL}" --no-cache-dir \
-      "${LINUX_REQUIREMENTS_TO_INSTALL[@]}"
-  )
-else
-  # For torchao need to install from github since nightly build doesn't have macos build.
-  # TODO: Remove this and install nightly build, once it supports macos
-  (
-    set -x
-    $PIP_EXECUTABLE install git+https://github.com/pytorch/ao.git@e11201a62669f582d81cdb33e031a07fb8dfc4f3
-  )
-fi
+(
+  set -x
+  $PIP_EXECUTABLE install torchao=="0.5.0"
```

Comment:
I think it's fine to have it here for now since we might move to nightlies, but this makes more sense in the install_requirements.txt